README.md

```diff
@@ -32,6 +32,8 @@ Refer to the table below for a detailed view of final models materialized by def
 | [app_reporting__overview_report](https://github.com/fivetran/dbt_app_reporting/blob/main/models/app_reporting__overview_report.sql)   | Each record represents daily metrics by app_name.                            |                          |
 | [app_reporting__traffic_source_report](https://github.com/fivetran/dbt_app_reporting/blob/main/models/app_reporting__traffic_source_report.sql) | Each record represents daily metrics by app_name and traffic source.                         |
 
+> The individual Google Play and Apple App Store models have additional platform-specific metrics better suited for deep-dive analyses.
```

Comment:
or something like this... i just wanna make sure people know there's more stuff in the upstream packages (though the DECISIONLOG alludes to this)